scripts/pre-commit.sh

```diff
@@ -22,14 +22,21 @@ set -o pipefail
 
 source "$(dirname "${BASH_SOURCE}")/util.sh"
 ROOT_DIR="$(cd "$(dirname "$0")/.." ; pwd)"
+TEMP_DIR="$(dirname $(mktemp -u))"
```

Comment:
A simpler way to find this would be `${TMPDIR}`.

But would it make more sense to create a new temporary directory with `mktemp -d` to ensure that a clean path was being used?